### PR TITLE
REAMDE.md

### PR DESCRIPTION
**Pull request by danny73210**
Please write down your bots for README.md

### CODE REVIEW

**Comment by Dinexpod** [comment]
Let's think, you need writing new README)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+**DESCRIPTION**
+
+PURPOSE
+
+This is Telegram Bot which takes users request to find some wiki-information
+using Telegram & Wiki API
```
Comment (boroda4436) [inline README.md]: Take in mind that we are not using wiki api anymore

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Using API of Telegram and Wikipedia to create telegram boot which process user request and finding most similar information on wikipedia
+
+AS USER
+
+As user u need to open your telegram application and find bot @WikiLookUpBot. Next u need to choose "look up" button to type your request, after that u will receive wiki-information
+
+AS DEVELOPER
+
+As developer u will need pc or laptop with Intelij Idea program on it and GitHub account. Next u need to 'fork' project and open in your Intelij Idea program and develope!
```
Comment (boroda4436) [inline README.md]: @danny73210 let's user be free in his choice to use any IDE
I'm not ok with this whole line, please remove it

File: README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+CONTRIBUTE
+
+Please, DO NOT PUSH ANY TOKEN OR API KEY, I will never accept a pull request with that content.
+
+Feel free to fork this project, work on it and then make a pull request against DEV branch. Most of the times I will accept them if they add something valuable to the code.
```
Comment (boroda4436) [inline README.md]: do we have DEV branch?

File: README.md (hunk 1 above)
Comment (Dinexpod) [reply README.md]: I mean this think)

**Comment by boroda4436** [comment]
@danny73210 add information about checkstyle workflow

**Comment by boroda4436** [comment]
@danny73210 add information how to open a PR correctly

**Comment by Dinexpod** [comment]
I don't see any changes

**Pull request closed without merging**